dds_web/api/api_s3_connector.py

```diff
@@ -84,11 +85,21 @@ def get_s3_info(self):
         )
 
     @bucket_must_exists
-    def remove_all(self, *args, **kwargs):
+    def remove_bucket(self, *args, **kwargs):
         """Removes all contents from the project specific s3 bucket."""
+        # Get bucket object
         bucket = self.resource.Bucket(self.project.bucket)
+
+        # Delete objects first
         bucket.objects.all().delete()
 
+        # Delete bucket
+        bucket.delete()
+
+        # Clean up
```

Comment:
Is this really needed?

Reply:
The clean up? It doesn't hurt, the bucket doesn't exist so there's no need to keep it in memory

Reply:
The effect should be the same if you just set it to None, or even just leave it.

Reply:
Yes, basically the same. So I will keep this. Plus this is the method we use in multiple other places in the code.